versioned_docs/version-2.0.0/keploy-explained/dev-guide.md

```diff
@@ -135,24 +135,21 @@ keployV2 test --c "docker run -p 8080:8080  --name <containerName> --network kep
 Run the below command inside the keploy respository and make sure there is no directory by the name of keploy inside the main keploy repository.
 
 ```shell
-docker build -t <nameOfImage> .
+sudo docker image build -t ghcr.io/keploy/keploy:v2-dev .
 ```
 
-#### Create Alias:
-
-````shell
-alias keployV2='sudo docker run --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v $(pwd):$(pwd) -w $(pwd) -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm <nameOfImage>'
```

Comment:
please keep the alias as it creates conflict if the user is already using keploy and also is setting up the dev-environment